.gitignore

```diff
@@ -0,0 +1,46 @@
+# Created by https://www.toptal.com/developers/gitignore/api/vim,go
```

Comment:
This .gitignore file looks very generic. I don't think it's a good fit for this project. I would suggest simply deleting it.

Reply:
It's generated from https://www.toptal.com/developers/gitignore/api/vim,go, where you put in the relevant languages and editors and it generates you a specific config. Half of it is specific to Golang projects. I'm not overly desperate to keep it, but I generated it to stop myself checking in VIM swap files.